targets/targets.json

```diff
@@ -7894,16 +7894,26 @@
     },
     "MCU_PSOC6_M4": {
         "inherits": ["MCU_PSOC6"],
+        "components_add": ["FLASHIAP"],
         "macros_add": ["MCU_PSOC6_M4"],
         "public": false
     },
+    "CY8CKIT_062_WIFI_BT_M0": {
+        "inherits": ["MCU_PSOC6_M0"],
+        "supported_form_factors": ["ARDUINO"],
+        "extra_labels_add": ["CY8C62XX", "CY8CKIT_062_WIFI_BT"],
+        "macros_add": ["CY8C6247BZI_D54", "PSOC6_DYNSRM_DISABLE=1"],
+        "detect_code": ["1900"],
+        "public": false
```

Comment:
missing `,` - explains the Travis failues

Reply:
Why are these set to false?

Reply:
We do not want the CM0 as a target directly. We only want to expose it in the context of what PSA requires.